### PR TITLE
Update mac-preferences.md

### LINKED ISSUE #12541: Update Intune Deployment section to the latest interface
Please update Intune deployment part to match the current Intune Set up name
Devices > Configuration Profiles > Create Profile > macOS > Templates > Custom
image

Current Docs is a little bit different:
image

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: 2dc758cd-0668-546f-5132-c3b5d2bf2b8e
* Version Independent ID: afa7ca04-3825-e73a-bd6e-4cf55010bd60
* Content: [Set preferences for Microsoft Defender for Endpoint on Mac](https://learn.microsoft.com/en-us/microsoft-365/security/defender-endpoint/mac-preferences?view=o365-worldwide#intune-full-profile)
* Content Source: [microsoft-365/security/defender-endpoint/mac-preferences.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/security/defender-endpoint/mac-preferences.md)
* Service: **microsoft-365-security**
* Sub-service: **mde**
* GitHub Login: @Dansimp
* Microsoft Alias: **dansimp**

### PR DESCRIPTION
**Pull request by msbemba**
made changes to the navigation

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs/issues/12541

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/mac-preferences.md [hunk 1]
```diff
@@ -1287,9 +1287,9 @@ From the JAMF console, open **Computers** \> **Configuration Profiles**, navigat
 
 ### Intune deployment
 
-1. Open **Manage** \> **Device configuration**. Select **Manage** \> **Profiles** \> **Create Profile**.
+1. Open **Devices** \> **Configration Profiles**. Select **Create Profile**.
 
-2. Choose a name for the profile. Change **Platform=macOS** to **Profile type=Custom**. Select Configure.
+2. Choose a name for the profile. Change **Platform=macOS** to **Profile type=Templates** and choose**Custom** in the template name section and Select Configure.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/mac-preferences.md]:
```suggestion
2. Choose a name for the profile. Change **Platform=macOS** to **Profile type=Templates** and choose **Custom** in the template name section. Select **Configure**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Dansimp Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@Dansimp Gentle reminder

**Pull request merged**